geotrek/trekking/models.py

```diff
@@ -516,7 +516,8 @@ class Practice(PictogramMixin):
     cirkwi = models.ForeignKey('cirkwi.CirkwiLocomotion', verbose_name=_(u"Cirkwi locomotion"), null=True, blank=True)
     order = models.IntegerField(verbose_name=_(u"Order"), null=True, blank=True, db_column='tri',
                                 help_text=_(u"Alphabetical order if blank"))
-    color = ColorField(verbose_name=_(u"Color (mobile app only)"), default='#444444', db_column='couleur')
+    color = ColorField(verbose_name=_(u"Color (mobile app only)"), default='#444444', db_column='couleur',
+                        help_text=_(u"Color of the practice, used in mobile and PDF export. (To be implemented in Geotrek-rando)"))
```

Comment:
"Color of the practice, used in mobile" c'est du paraphrasage donc pas utile. Sur le PDF, je pense qu'il ne faut pas rendre cela générique. Ok pour l'utiliser de manière spécifique pour le PNC si ils le demandent, mais pas pour tout le monde.

Reply:
Du coup je propose de mettre "Color" pour le nom du champs, et "Color of the practice, only used in mobile. (To be implemented in Geotrek-rando)" pour le message d'aide, comme ça pas de paraphrase mais ça reste logique.